src/uniparc/components/entry/XRefsSection.tsx

```diff
@@ -0,0 +1,226 @@
+import { FC, ReactNode, useMemo } from 'react';
+import { Link, generatePath } from 'react-router-dom';
+import { Card, DataTable, ExternalLink } from 'franklin-sites';
+
+import CustomiseButton from '../../../shared/components/action-buttons/CustomiseButton';
+import { EntryTypeIcon } from '../../../shared/components/entry/EntryTypeIcon';
+
+import useStaggeredRenderingHelper from '../../../shared/hooks/useStaggeredRenderingHelper';
+import useDataApi from '../../../shared/hooks/useDataApi';
+import useLocalStorage from '../../../shared/hooks/useLocalStorage';
+
+import { defaultColumns } from '../../config/UniParcColumnConfiguration';
+
+import apiUrls from '../../../shared/config/apiUrls';
+
+import { Location, LocationToPath } from '../../../app/config/urls';
+import { Namespace } from '../../../shared/types/namespaces';
+import { Column } from '../../../shared/config/columns';
+import { UniParcXRef } from '../../adapters/uniParcConverter';
+import EntrySection, {
+  getEntrySectionNameAndId,
+} from '../../types/entrySection';
+import { EntryType } from '../../../uniprotkb/adapters/uniProtkbConverter';
+
+import './styles/XRefsSection.scss';
+
+const getColumns = (
```

Comment:
I guess this will moved to a column config map after the API model changes?

Reply:
Where this piece of logic is located doesn't really depend on the API model change. If we want that in another file we can do it.
I just followed the same approach that we have in other data tables within sections which is collocate the column configuration with the `<DataTable />` component in the same file (as it's not going to be used anywhere else).
See UniProtKB's SimilarProteinsTable and ComputationallyMappedSequences and UniRef's MembersSections

Reply:
Difference being that this one is customisable whereas the others are not

Reply:
So this one should stay and the others should move to distinct files?
Also, the one in ComputationallyMappedSequences should be outside of the render function anyway.

Reply:
Yep you're right about `ComputationallyMappedSequences` will update that.

What I meant was that because it's selectable columns, it might be better to use the same definition model used in `UniRefColumnConfiguration` and other namespaces. `UniRefEntryColumnConfiguration`  for instance.

Reply:
Ah I see.
Will do that after the schema update then. Not sure yet if I will get a configure endpoint specific to UniParc entries (different than for UniParc results) or if I'll have to do that on the client somehow. The current column selection component assumes that only result page configuration endpoints exists and automatically loads it so I'd have to override that.
That needs to be done at a later point

Reply:
Looks like this will be next sprint so unlikely we'll forget 🤞